SamplesDashboard/SamplesDashboard/ClientApp/src/views/Details.tsx

```diff
@@ -41,23 +42,32 @@ export default class Details extends React.Component<any, any> {
         const repositoryName = params.name;
         const response = await fetch('api/samples/' + repositoryName);
         const data = await response.json();
-        this.allItems = data;
+        if (data.dependencyGraphManifests.nodes[0]) {
+            this.allItems = data.dependencyGraphManifests.nodes[0].dependencies.nodes;
+        }
         this.setState({
             items: this.allItems,
             repositoryDetails: {
-                name: repositoryName
+                name: repositoryName,
+                url: data.url
             },
             isLoading: false
         });
     }    
     
 
     public render(): JSX.Element {
-        const { columns, items, repositoryDetails,isLoading } = this.state;
+        const { columns, items, repositoryDetails, isLoading } = this.state;
+        const repoName = repositoryDetails.name;
+        
         return (
             <Fabric>
                 <div>
-                    <PageTitle title={repositoryDetails.name} />
+                    <PageTitle title={repoName} />
```

Comment:
>repoName [](start = 38, length = 8)

NIT: seems a little inconsistent that you are using repositoryDetails.url below but here you've assigned a variable that's really only ever used in this case.